web/src/app/details/DetailsPage.js

```diff
@@ -185,10 +191,13 @@ export default function DetailsPage(props) {
     </Grid>
   )
 }
+
 DetailsPage.propTypes = {
   title: p.string,
   details: p.string,
 
+  notices: p.array, // todo: use type Notice when converted to ts
```

Comment:
Define the full type here
```suggestion
  notices: p.arrayOf(p.shape({/* ... */}), // todo: use type Notice when converted to ts
```